reana_client/cli/workflow.py

```diff
@@ -868,6 +868,7 @@ def interactive_group():
 @click.argument(
     'interactive-session-type',
     metavar='interactive-session-type',
+    default=INTERACTIVE_SESSION_TYPES[0],
```

Comment:
Since `close` closes jupyter, but default, I guess we can go with it, but there might be changes ahead later.